Documentation/ceph-filesystem-crd.md

```diff
@@ -94,8 +96,7 @@ spec:
     activeStandby: true
 ```
 
-(These definitions can also be found in the [`filesystem-ec.yaml`](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/filesystem-ec.yaml) file.
-Also see an example in the [`storageclass-ec.yaml`](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph/csi/cephfs/storageclass-ec.yaml) for how to configure the volume.)
+**IMPORTANT**: For erasure coded pools, we have to create a replicated pool as the default data pool and an erasure-coded pool as a secondary pool.
```

Comment:
Could we get a confirmation from the cephfs team about the setattr that was suggested before? It isn't needed?

Reply:
AFAIU, if you want a CephFS's dir tree data to be stored in a specific data pool (an ec pool in this scenario), then you'd need to set the dir.layout.pool attribute as done here,  https://docs.ceph.com/en/latest/cephfs/file-layouts/#adding-a-data-pool-to-the-file-system

@batrick @kotreshhr ^^

Reply:
This is what we wanted to confirm as while testing I didn't run any command for file layout and we can see [here](https://github.com/rook/rook/pull/8452#issuecomment-977690746) data is in the erasure-coded pool.